Pages/Energy.cs

```diff
@@ -2,23 +2,21 @@
 {
     using System;
     using System.Collections.Generic;
-    using System.Linq;
     using System.Threading.Tasks;
-
     using System.Globalization;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
-
+  
     public partial class Energy
     {
-        [JsonProperty("data_year")]
+        [JsonProperty("dataYear")]
```

Comment:
This string should match the JSON source data, and use the naming conventions in that JSON source data.  I have a hunch that changing the naming that we see here would not work.